video/out/w32_common.c

```diff
@@ -1543,6 +1582,10 @@ static int gui_thread_control(struct vo_w32_state *w32, int request, void *arg)
         if (w32->opts->fullscreen != w32->current_fs)
             reinit_window_state(w32);
         return VO_TRUE;
+    case VOCTRL_GET_FULLSCREEN:
+        *(int *)arg = w32->toggle_fs ^ w32->current_fs;
+        reinit_window_state(w32);
```

Comment:
Depends what the intention is. VOCTRL_GET_FULLSCREEN is only for copying the real fullscreen state to the frontend, if it has changed on its own in the win32 thread. To which line would you move the call?